static/src/javascripts/projects/common/modules/identity/ad-prefs.lib.spec.js

```diff
@@ -0,0 +1,51 @@
+// @flow
+import { getCookie as getCookie_, addCookie as addCookie_ } from 'lib/cookies';
+import { setAdConsentState, getAdConsentState } from './ad-prefs.lib';
+import type { AdConsent } from './ad-prefs.lib';
+
+const getCookie: any = getCookie_;
+const addCookie: any = addCookie_;
+
+const testConsent: AdConsent = {
+    label: 'Test consent',
+    cookie: 'GU_AD_CONSENT_TEST',
+};
+
+jest.mock('lib/cookies', () => ({
+    getCookie: jest.fn(() => null),
+    addCookie: jest.fn(() => null),
+}));
+
+beforeEach(() => {
+    addCookie.mockReset();
+    getCookie.mockReset();
+});
+
+describe('getAdConsentState', () => {
+    it('should convert false & true cookies properly', () => {
+        getCookie.mockImplementation(() => '0');
+        expect(getAdConsentState(testConsent)).toBe(false);
+        getCookie.mockImplementation(() => '1');
+        expect(getAdConsentState(testConsent)).toBe(true);
+    });
+    it('should convert null & inconsistent cookies properly', () => {
+        getCookie.mockImplementation(() => null);
+        expect(getAdConsentState(testConsent)).toBe(null);
+        getCookie.mockImplementation(() => 'verdadero');
+        expect(getAdConsentState(testConsent)).toBe(null);
+    });
+});
+
+describe('setAdConsentState', () => {
+    it('should set a full proper cookie', () => {
+        setAdConsentState(testConsent, true);
+        expect(addCookie.mock.calls[0][0]).toMatch('GU_AD_CONSENT_TEST');
+        expect(addCookie.mock.calls[0][1]).toMatch('1');
+        expect(addCookie.mock.calls[0][2]).toBeGreaterThanOrEqual(365 * 4);
```

Comment:
I see that this would pass, but I was wondering why have the expectation as `toBeGreaterThanOrEqual(365 * 4)` and not `toBe(365 * 6)` as this `356 * 6` is hardcoded in `setAdConsentState`?